icu-date/ffi_cdef.lua

```diff
@@ -252,6 +252,8 @@ return function(ffi, icu_version_suffix)
       UCAL_DECEMBER,
       UCAL_UNDECIMBER
     } UCalendarMonths;
+    typedef uint8_t UVersionInfo[20];
```

Comment:
> The binary form of a version on ICU APIs is an array of 4 uint8_t. 

https://unicode-org.github.io/icu-docs/apidoc/dev/icu4c/uversion_8h.html#a61caef96ba86d8e86a4031925ea44e94

Here we have 20 bytes and I don't understand from where this number comes.

Reply:
It's my bad. I took U_MAX_VERSION_STRING_LENGTH value instead of U_MAX_VERSION_LENGTH. Fixed. Thanks.